### ISSUE TITLE
Google Spreadsheet integration error

### ISSUE THREAD

**Issue by andypullen** [verbb/formie#1175]
### Describe the bug

Formie is returns errors when accessing an existing Google Spreadsheet integration

verbb\formie\errors\IntegrationException in POST /index.php
API error: “verbb\formie\integrations\miscellaneous\GoogleSheets::getProxyRedirect(): Return value must be of type string, null returned” /var/www/craft/vendor/verbb/formie/src/integrations/miscellaneous/GoogleSheets.php:90

`/var/www/craft/vendor/verbb/formie/src/base/Integration.php:174 - verbb\formie\base\Integration::apiError`

### Steps to reproduce

With an existing Google Spreadsheet integration, clicking into either Formie's integration settings or refreshing the available tables on the spreadsheet from within a form's integration settings, the above exception gets triggered.

### Form settings

Google spreadsheet integration is needed

### Craft CMS version

3.7.59

### Plugin version

1.6.18

### Additional context

This happens on both a production and dev environment

**Comment by andypullen** [comment]
Here is the full stacktrace

```
TypeError verbb\formie\integrations\miscellaneous\GoogleSheets::getProxyRedirect(): Return value must be of type string, null returned 
    /var/www/craft/vendor/verbb/formie/src/integrations/miscellaneous/GoogleSheets.php:90 verbb\formie\integrations\miscellaneous\GoogleSheets::getProxyRedirect
    /var/www/craft/vendor/verbb/formie/src/integrations/miscellaneous/GoogleSheets.php:112 verbb\formie\integrations\miscellaneous\GoogleSheets::getRedirectUri
    /var/www/craft/vendor/twig/twig/src/Extension/CoreExtension.php:1566 twig_get_attribute
    /var/www/craft/vendor/craftcms/cms/src/helpers/Template.php:106 craft\helpers\Template::attribute
    /var/www/craft/storage/runtime/compiled_templates/a5/a53033bf54a4846cb79f0e74b61465ddb04974b2eab84a52503ff6c9044c657f.php:71 __TwigTemplate_ef63841650a6118fd3e1d8b465b5a93774874c9dcec9d710260999b48e980a0d::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/vendor/twig/twig/src/Template.php:390 Twig\Template::render
    /var/www/craft/vendor/twig/twig/src/TemplateWrapper.php:45 Twig\TemplateWrapper::render
    /var/www/craft/vendor/twig/twig/src/Environment.php:318 Twig\Environment::render
    /var/www/craft/vendor/craftcms/cms/src/web/View.php:408 craft\web\View::renderTemplate
    /var/www/craft/vendor/verbb/formie/src/base/Miscellaneous.php:58 verbb\formie\base\Miscellaneous::getSettingsHtml
    /var/www/craft/vendor/twig/twig/src/Extension/CoreExtension.php:1566 twig_get_attribute
    /var/www/craft/vendor/craftcms/cms/src/helpers/Template.php:106 craft\helpers\Template::attribute
    /var/www/craft/storage/runtime/compiled_templates/50/50e87679d211c1bf339ea9fa404c248a70bbe1f84445833e3e7ad8a3f78cc834.php:215 __TwigTemplate_226effadb7528fca97ac60026be7e47d75d545cf4e9739482f418adfd64cf55f::block_content
    /var/www/craft/vendor/twig/twig/src/Template.php:182 Twig\Template::displayBlock
    /var/www/craft/storage/runtime/compiled_templates/a2/a2938622a1e03013b96b6173fae2409faa8603ab0b29d82d551f70818173823d.php:547 __TwigTemplate_3d4b34344e9303d33c9dd14aaafad28387450555fa231e6ddac1866f75043069::block_main
    /var/www/craft/vendor/twig/twig/src/Template.php:182 Twig\Template::displayBlock
    /var/www/craft/storage/runtime/compiled_templates/a2/a2938622a1e03013b96b6173fae2409faa8603ab0b29d82d551f70818173823d.php:358 __TwigTemplate_3d4b34344e9303d33c9dd14aaafad28387450555fa231e6ddac1866f75043069::block_body
    /var/www/craft/vendor/twig/twig/src/Template.php:182 Twig\Template::displayBlock
    /var/www/craft/storage/runtime/compiled_templates/70/70a0b6d453d8b5b0e9a99372f8601f5047cf4b5a6b2d8b9393cb8acb3425bf9c.php:91 __TwigTemplate_bdecb6bf891c2d758a99924cca8b5bd9d63b4504cbfb280db825fd92f6675e74::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/storage/runtime/compiled_templates/58/5835be29f7691c417fbcb234196c43661f2d97bb89b01191bf26a1f743123f82.php:56 __TwigTemplate_07f1a1db122993ac081a3c73690f0a733586d171ae4916cecce5e2a0b9aebe4b::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/storage/runtime/compiled_templates/a2/a2938622a1e03013b96b6173fae2409faa8603ab0b29d82d551f70818173823d.php:172 __TwigTemplate_3d4b34344e9303d33c9dd14aaafad28387450555fa231e6ddac1866f75043069::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/storage/runtime/compiled_templates/12/120384d6cce084a0f406e2ef64e222e2b44bd031a162a00ac9d1f02ae36d8d87.php:87 __TwigTemplate_31f256a5509f2433f2ae24555e6b53794f0b9197113576d2956f379e821c5b82::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/storage/runtime/compiled_templates/50/50e87679d211c1bf339ea9fa404c248a70bbe1f84445833e3e7ad8a3f78cc834.php:104 __TwigTemplate_226effadb7528fca97ac60026be7e47d75d545cf4e9739482f418adfd64cf55f::doDisplay
    /var/www/craft/vendor/twig/twig/src/Template.php:405 Twig\Template::displayWithErrorHandling
    /var/www/craft/vendor/twig/twig/src/Template.php:378 Twig\Template::display
    /var/www/craft/vendor/twig/twig/src/Template.php:390 Twig\Template::render
    /var/www/craft/vendor/twig/twig/src/TemplateWrapper.php:45 Twig\TemplateWrapper::render
    /var/www/craft/vendor/twig/twig/src/Environment.php:318 Twig\Environment::render
    /var/www/craft/vendor/craftcms/cms/src/web/View.php:408 craft\web\View::renderTemplate
    /var/www/craft/vendor/craftcms/cms/src/web/View.php:461 craft\web\View::renderPageTemplate
    /var/www/craft/vendor/craftcms/cms/src/web/Controller.php:205 craft\web\Controller::renderTemplate
    /var/www/craft/vendor/verbb/formie/src/controllers/IntegrationSettingsController.php:283 verbb\formie\controllers\IntegrationSettingsController::_editIntegration
    /var/www/craft/vendor/verbb/formie/src/controllers/IntegrationSettingsController.php:206 verbb\formie\controllers\IntegrationSettingsController::actionEditMiscellaneous
    [internal] call_user_func_array
    /var/www/craft/vendor/yiisoft/yii2/base/InlineAction.php:57 yii\base\InlineAction::runWithParams
    /var/www/craft/vendor/yiisoft/yii2/base/Controller.php:178 yii\base\Controller::runAction
    /var/www/craft/vendor/yiisoft/yii2/base/Module.php:552 yii\base\Module::runAction
    /var/www/craft/vendor/craftcms/cms/src/web/Application.php:293 craft\web\Application::runAction
    /var/www/craft/vendor/yiisoft/yii2/web/Application.php:103 yii\web\Application::handleRequest
    /var/www/craft/vendor/craftcms/cms/src/web/Application.php:278 craft\web\Application::handleRequest
    /var/www/craft/vendor/yiisoft/yii2/base/Application.php:384 yii\base\Application::run
    /var/www/craft/web/index.php:21 [main]
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.18](https://github.com/verbb/formie/releases/tag/2.0.18) and [1.6.19](https://github.com/verbb/formie/releases/tag/1.6.19)
